src/web/Router.js

```diff
@@ -9,6 +9,7 @@ import Footer from "./theme/modules/Footer";
 import Home from "./theme/pages/Home";
 import Product from "./theme/pages/Product";
 import Checkout from "./theme/pages/Checkout";
+import TryItPipe from "./theme/pages/TryItPipe";
```

Comment:
I don't understand the term "Pipe" in the name of this, care to explain ? :smiley:

Reply:
i thought about a "tunnel" .. but i'll just remove it it's not necessary